homeassistant/components/maxcube/__init__.py

```diff
@@ -23,12 +22,14 @@
 NOTIFICATION_TITLE = "Max!Cube gateway setup"
 
 CONF_GATEWAYS = "gateways"
+CONF_SHARED = "shared"
 
 CONFIG_GATEWAY = vol.Schema(
     {
         vol.Required(CONF_HOST): cv.string,
         vol.Optional(CONF_PORT, default=DEFAULT_PORT): cv.port,
-        vol.Optional(CONF_SCAN_INTERVAL, default=300): cv.time_period,
+        vol.Optional(CONF_SCAN_INTERVAL, default=0): cv.time_period,
+        vol.Optional(CONF_SHARED, default=False): cv.boolean,
```

Comment:
We no longer accept YAML configuration for integrations that connect to devices or services.

Instead, a configuration flow via the UI should be used.

For more information about this decision see:

- ADR-0010: <https://github.com/home-assistant/architecture/blob/master/adr/0010-integration-configuration.md#decision>

To get started on the integration configuration, see our developer documentation:

<https://developers.home-assistant.io/docs/config_entries_config_flow_handler>

Reply:
Would you accept the bugfix without the configuration changes?

Reply:
After reading the documentation, it seems that I need to use the async API to use config flows. Does it also affect the rest of the component? Should I migrate the full component to async API?

Reply:
No, an async API/integration is not a requirement for having configuration flows. The configuration flow itself is async, but the rest can just be sync.